src/Tizen.NET.MaterialComponents/Components/MConfirmationDialog.cs

```diff
@@ -0,0 +1,233 @@
+    {
+        readonly int MaximumHeight = 500;
+        readonly int ItemHeight = 120;
+
+        IDictionary<int, MConfirmationDialogItem> _itemList = new Dictionary<int, MConfirmationDialogItem>();
+        Button _confirmButton;
+        Button _cancelButton;
+        Box _layout;
+        GenList _genList;
+        GenItemClass _itemClass;
+        int _itemIndex = 0;
+        Color _defaultTextColor;
+        Color _defaultTextColorForDisable;
+
+        public MConfirmationDialog(EvasObject parent) : base(parent)
+        {
+            Style = Styles.Popup.Confirmation;
+
+            _layout = new Box(parent)
+            {
+                AlignmentX = -1,
+                AlignmentY = -1,
+                WeightX = 1,
+                WeightY = 1,
+            };
+            _layout.Show();
+
+            _genList = new GenList(parent)
+            {
+                WeightX = 1,
+                WeightY = 1,
+                AlignmentY = -1,
+                AlignmentX = -1,
+                SelectionMode = GenItemSelectionMode.Always
+            };
+
+            _itemClass = new GenItemClass(Styles.GenListItem.Default)
+            {
+                GetContentHandler = GetContent,
+                GetTextHandler = GetText
+            };
+
+            _genList.ItemSelected += (s, e) =>
+            {
+                var item = e.Item.Data as MConfirmationDialogItem;
+                if (item.CheckBox is Radio r)
+                {
+                    r.GroupValue = item.Id;
+                }
+                else if (item.CheckBox is Check c)
+                {
+                    c.IsChecked = !c.IsChecked;
+                }
+
+                _confirmButton.IsEnabled = (_itemList.Where(kv => kv.Value.IsSelected).Count() > 0);
+            };
+
+            OutsideClicked += (s, e) =>
+            {
+                Dismiss();
+            };
+
+            _genList.Show();
+            _layout.PackEnd(_genList);
+
+            _confirmButton = new Button(this)
+            {
+                Style = Styles.Popup.PopupButton,
+                IsEnabled = false
+            };
+
+            _confirmButton.Clicked += (s, e) =>
+            {
+                Dismiss();
+                var selected = _itemList.Where(kv => kv.Value.IsSelected).Select(kv => kv.Value);
+                ItemSelected?.Invoke(this, new MDialogItemSelectedArgs(selected));
+            };
+
+            _cancelButton = new Button(this)
+            {
+                Style = Styles.Popup.PopupButton
+            };
+
+            _cancelButton.Clicked += (s, e) =>
+            {
+                Dismiss();
+            };
+
+            SetPartContent(Parts.Popup.Button2, _confirmButton);
+            SetPartContent(Parts.Popup.Button1, _cancelButton);
+
+            SetContent(_layout);
+
+            MColors.AddColorSchemeComponent(this);
+        }
+
+        public event EventHandler<MDialogItemSelectedArgs> ItemSelected;
```

Comment:
Why not `MConfirmationDialogItemSelectedArgs`?

Reply:
Done.